server/flyway/sql/V20__fix_timestamp_to_timestamptz.sql

```diff
@@ -0,0 +1,80 @@
+-- Running upgrade V19 -> V20
+
+-- Change date type to timestamptz for create_date and update_date
```

Comment:
This is probably extra, or maybe no need to do it. From my understanding from Basil's safe concerns, would it be safer to also set postgres database timezone to UTC (as using "ALTER DATABASE...")?

Reply:
Database should already default to UTC.

Reply:
yeah, think we discussed this yesterday, for best practice, change to use timestamptz, so it's ensured and always come up with a timezone.